src/Core/Addon/Theme/Theme.php

```diff
@@ -51,11 +51,11 @@ public function __construct(
     ) {
         if (isset($attributes['parent'])) {
             if (null === $configurationCacheDirectory) {
-                $configurationCacheDirectory = (new Configuration())->get('_PS_CACHE_DIR_');
+                $configurationCacheDirectory = _PS_CACHE_DIR_;
             }
 
             $yamlParser = new YamlParser($configurationCacheDirectory);
-            $parentAttributes = $yamlParser->parse($themesDirectory . '/' . $attributes['parent'] . '/config/theme.yml');
+            $parentAttributes = $yamlParser->parse($themesDirectory . $attributes['parent'] . '/config/theme.yml');
```

Comment:
It is possible the slash is needed for top-level themes and not needed for child themes, this might be why CI is failing